annotations/templates/annotations/concept_add.html

```diff
@@ -77,7 +77,7 @@
     </ul>
     {% endif %}
 
-    <a class="btn btn-md btn-danger" href="{{ next_page }}">Take me back!</a>
+    <input class="btn btn-md btn-danger" type=button value="Take me back!" onClick="javascript:history.go(-1);">
```

Comment:
let's not use the browser history but actually fix what ever was wrong with the next_page link